### DISCUSSION TITLE
I always get this error. I reinstalled the Nvidia graphics card driver and still set export OMNIGIBSON_GPU-ID=0, but it still doesn't work.OmniGraphSettings::getCudaDeviceOrdinal: unable to get a valid CUDA device id from the renderer. Defaulting to GPU0.

### DISCUSSION THREAD

**Discussion by vito9394** [StanfordVL/BEHAVIOR-1K#1030] [Q&A]
**PLEASE POST QUESTIONS IN THE DISCUSSIONS TAB**
nvidia-smi  and nvcc -V ，Please reply as soon as possible. It's very urgent

![屏幕截图 2024-11-05 103430](https://github.com/user-attachments/assets/f9788f4d-4abb-4f3f-9ec7-1bb6588a455b)

**Comment by hang-yin** [comment]
@cgokmen probably has a better answer to this

**Comment by cgokmen** [comment]
Hi, please provide more information about your setup. Are you running this inside WSL? Isaac Sim does not support that (Vulkan access is not available on WSL)
